src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/Services/PermalinkServiceTests.cs

```diff
@@ -317,71 +289,47 @@ public async Task Get()
                 blobText: JsonConvert.SerializeObject(permalink));
 
-            subjectRepository
-                .Setup(s => s.Get(subject.Id))
-                .ReturnsAsync(subject);
-
-            releaseRepository
-                .Setup(s => s.IsLatestVersionOfRelease(release.PublicationId, release.Id))
-                .Returns(true);
-
-            var statisticsDbContextId = Guid.NewGuid().ToString();
-            await using (var statisticsDbContext = StatisticsDbUtils.InMemoryStatisticsDbContext(statisticsDbContextId))
-            {
-                await statisticsDbContext.AddAsync(new ReleaseSubject
-                {
-                    Release = release,
-                    Subject = subject,
-                });
-
-                await statisticsDbContext.SaveChangesAsync();
-            }
-
             var contentDbContextId = Guid.NewGuid().ToString();
             await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
             {
-                await contentDbContext.AddAsync(
-                    new Content.Model.Release
+                await contentDbContext.AddRangeAsync(
+                    release,
+                    new ReleaseFile
                     {
-                        Id = release.Id,
-                        Publication = new Content.Model.Publication { Id = _publicationId },
+                        Release = release,
+                        File = new File
+                        {
+                            SubjectId = subjectId,
+                            Type = FileType.Data,
+                        },
                     });
 
                 await contentDbContext.SaveChangesAsync();
             }
 
             await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
-            await using (var statisticsDbContext = StatisticsDbUtils.InMemoryStatisticsDbContext(statisticsDbContextId))
             {
                 var service = BuildService(
                     contentDbContext: contentDbContext,
-                    statisticsDbContext: statisticsDbContext,
-                    blobStorageService: blobStorageService.Object,
-                    subjectRepository: subjectRepository.Object,
-                    releaseRepository: releaseRepository.Object);
+                    blobStorageService: blobStorageService.Object);
 
                 var result = (await service.Get(permalink.Id)).AssertRight();
 
-                MockUtils.VerifyAllMocks(
-                    blobStorageService,
-                    subjectRepository,
-                    releaseRepository);
+                MockUtils.VerifyAllMocks(blobStorageService);
 
                 Assert.Equal(permalink.Id, result.Id);
                 Assert.InRange(DateTime.UtcNow.Subtract(result.Created).Milliseconds, 0, 1500);
-                Assert.False(result.Invalidated);
+                Assert.Equal(PermalinkStatus.Current, result.Status);
```

Comment:
I think it would be worth either amending this test or adding another variation which checks the status is Current when the subject is linked through ReleaseFiles to multiple versions of the release.